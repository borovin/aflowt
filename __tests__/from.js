const Browser = require('../tools/browser');

let browser;

beforeEach(() => {
    browser = new Browser();
    browser.goto('/examples/form.html');
});

afterEach(() => {
    browser.close();
});

test('Primary button example', async () => {
    const snapshot = await browser.snapshot('b-form');

    expect(snapshot).toMatchSnapshot();

    browser.close();
});
