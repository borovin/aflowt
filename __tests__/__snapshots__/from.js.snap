// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Primary button example 1`] = `
<b-form action="/examples/form.html">
  <form>
    <b-input name="A" label="A field" type="text"></b-input>
    <b-input name="B" label="B field" value="default B value" type="text"></b-input>
    <b-button color="primary">Save</b-button>
  </form>
</b-form>
`;
